docs/setup_guide.md

```diff
@@ -0,0 +1,120 @@
+# SafeSnap Setup Guide
+
+This guide shows how to setup the DAO module with a Gnosis Safe on the Rinkeby testnetwork. It will use [Realitio](https://realit.io/) and can be used with [Snapshot](https://snapshot.org/).
+
+For more information on SafeSnap please refer to the [Gnosis blog](https://blog.gnosis.pm/ea67eb95c34f).
```

Comment:
I understand `SafeSnap` to be the name for the entire concept/infrastructure (frontend/backend/contract) allowing users in a DAO to execute successful proposals onchain. I think this should be stated somewhere in the main readme, since I came here with the intention of `Follow our [setup guide](./docs/setup_guide.md) to get started with the DAO module` but I got a guide that explains how to set up a mysterious `SafeSnap`.

Reply:
Yes, I can see why the naming is a little confusing.
SafeSnap is the name of the specific configuration using a Gnosis Safe, the dao module, and the corresponding Snapshot plugin. We should probably make that more explicit in the link from the readme.

Reply:
I adjusted the link in the readme to make this more clear